.git-blame-ignore-revs

```diff
@@ -0,0 +1,2 @@
+# Migrated to prettier for code formatting
+e37e1ee5ec358398863a74f37bed197788ca8ecf
```

Comment:
what purpose does this serve?

Reply:
Remember the 600 files I changed in the last PR? This ensures that all lines changed in that commit will be ignored when using git-blame (on GitHub) and locally (but you do need to configure that locally, if you want to use it). Otherwise, git-blame will show that I wrote 90% of the frontend which is not helpful when you are trying to figure out who wrote a specific bit of code.

Reply:
ah okay